docs/api/quickstart_sinead.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+---
+
+# Quickstart guide
+
+Have you a lot on your plate to keep track of?
+
+Easily manage all your tasks in one simple to-do list!
+
+## At a glance
+
+This Quickstart provides all the information you need to begin using To-Do to track your tasks.
+
+You’ll learn when and how to use the web service and get set up to make your first call to the API.
+
+## Setting up the To-Do service
+
+Tasks will surface in the To-Do service when they are created in the database and assigned to users (by user {id}). If you’ve used the To-Do service previously, you’re already enrolled in the service and may have tasks to work with.
+
+If not, you might need to set up your development system and get going from scratch. Don’t worry – you only have to do this one time per development system! Follow these [prerequisite steps](https://uwc2-apidoc.github.io/to-do-service-public/tutorials/before-you-start-a-tutorial.html) to install the tools and test your development system.
+
+## When to use the To-Do service
+
+The To-Do service REST API offers a wide range of integration possibilities, from enhancing internal workflow to creating customer-facing task reminder apps.
+
+The service comprises two resources: [`user`](user) and [`task`](task). The [`user`](user) resource (containing the subscribers to the service) works in synergy with the [`task`](task) resource (containing the tasks) to align users with their to-do list reminders.
+
+Using this cloud-based service, customers can register themselves to create and manage their own tasks.
+When they're registered, customers can update and delete tasks to suit their needs. Adding tasks on customers' behalf is easy - if they're collaborating with colleagues on different teams and projects, for example, they might be delegated requests and assigned tasks from different sources.
+
+## How to use the To-Do service
+
+To build your API call, you must have the following components:
+
+* **A host.**  The {base_url} depends on users' installation of the service in their development environment. For v1 of To-Do Service API, the **base_url** variable is typically set to `http://localhost:3000`.
+* **Authorization.**  For v1 of the To-Do service, requests do not use any authorization. All endpoints are available to all users and applications.
+* **A request.**  The To-Do service REST API enables CRUD operations via HTTP requests on database resources (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE` methods). Request and response bodies are encoded as JSON.
+
+### Supported endpoints
+
+| HTTP Method | Endpoint |
+| :--------------: | :--------------: |
+| GET | [List all users](https://uwc2-apidoc.github.io/to-do-service-public/api/users-get-all-users) |
```

Comment:
```suggestion
| GET | [List all users](users-get-all-users.md) |
```
Is there a reason you used the complete URL in the links? Normally, for topics in the documentation set, a relative link (such as the suggestion) is preferred for a variety of reasons, such as: it's shorter, the IDE can validate it's existence, the web server can optimize its delivery.

If a relative link will work, can you update the link targets, please?
Please update the link